third_party/prost/Cargo.toml

```diff
@@ -48,14 +47,11 @@ prost-derive = { version = "0.8.0", path = "prost-derive", optional = true }
 
 [dev-dependencies]
 criterion = "0.3"
-env_logger = { version = "0.8", default-features = false }
-log = "0.4"
 proptest = "1"
 rand = "0.8"
 
-[profile.bench]
-debug = true
-
+# [profile.bench]
```

Comment:
Why did we comment this out?

Reply:
Because it's a workspace-specific value, removed entirely now.